src/caretogether-pwa/src/Referrals/Arrangements/ArrangementsSection/ArrangementsSection.tsx

```diff
@@ -0,0 +1,165 @@
+import { Masonry } from '@mui/lab';
+import AddCircleIcon from '@mui/icons-material/AddCircle';
+import { Grid, Typography, Button, useTheme } from '@mui/material';
+import { Box, useMediaQuery } from '@mui/system';
+import {
+  ArrangementPolicy,
+  CombinedFamilyInfo,
+  Permission,
+  Referral,
+} from '../../../GeneratedClient';
+import { ArrangementCard } from '../ArrangementCard';
+import { ToggleButton, ToggleButtonGroup } from '@mui/material';
+import { CreateArrangementDialog } from '../CreateArrangementDialog';
+import { useRef, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { policyData } from '../../../Model/ConfigurationModel';
+import { getFilteredArrangements } from './getFilteredArrangements';
+import { useScrollToArrangement } from './useScrollToArrangement';
+
+type ArrangementSectionProps = {
+  referral: Referral;
+  family: CombinedFamilyInfo;
+  permissions: (permission: Permission) => boolean;
+};
+
+export function ArrangementsSection({
+  referral,
+  family,
+  permissions,
+}: ArrangementSectionProps) {
+  const [selectedFilters, setSelectedFilters] = useState<string[]>([
+    'Active',
+    'Ended',
+    'Cancelled',
+  ]);
+
+  const policy = useRecoilValue(policyData);
+
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up('sm'));
+  const isWideScreen = useMediaQuery(theme.breakpoints.up('xl'));
+
+  const [
+    createArrangementDialogParameter,
+    setCreateArrangementDialogParameter,
+  ] = useState<ArrangementPolicy | null>(null);
+
+  const filteredArrangements = getFilteredArrangements(
+    referral,
+    selectedFilters
+  );
+
+  const arrangementRefs = useRef<Record<string, HTMLDivElement | null>>({});
+
+  useScrollToArrangement(arrangementRefs);
+
+  return (
+    <Grid item xs={12}>
+      <div
+        style={{
+          display: `flex`,
+          justifyContent: `space-between`,
+          maxWidth: `100%`,
+          flexWrap: `wrap`,
+        }}
+      >
+        <div
+          style={{
+            display: `flex`,
+            justifyContent: `flex-start`,
+            maxWidth: `100%`,
+            flexWrap: `wrap`,
+          }}
+        >
+          <Box
+            display="flex"
+            alignItems="center"
+            justifyContent="space-between"
+            flexWrap="wrap"
+            gap={2}
+            mb={2}
+          >
+            <Typography
+              className="ph-unmask"
+              variant="h3"
+              sx={{ m: 0, display: 'flex', alignItems: 'center' }}
+            >
+              Arrangements
+            </Typography>
+
+            <ToggleButtonGroup
+              value={selectedFilters}
```

Comment:
The only change I would make is add `  color="primary"`, I think it make the togglebutton visual more consistent with the app
```suggestion
              color="primary"
              value={selectedFilters}
```